### PR TITLE
[WFLY-6460] Add analysis documentation for JSON-P 1.1 (JSR 374).

### PR DESCRIPTION
**Pull request by jamezp**

### CODE REVIEW

File: ee/WFLY-6460_JSON-P_1.1.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
+= Title
```
Comment (jamezp) [inline ee/WFLY-6460_JSON-P_1.1.adoc]: I'll fix the title after I'm finish my current changes to another document.

File: ee/WFLY-6460_JSON-P_1.1.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+* WildFly Core
+* WildFly
+
+== Requirements
+
+Add conditional support for JSON-P 1.1. This will be controlled by the `ee8.preview.mode` system property. If
+the property is not present or not set to `true` JSON-P 1.0 will be used instead.
+
+There is a known issue however with setting the `ee8.preview.mode` system property in the configuration file.
```
Comment (kabir) [inline ee/WFLY-6460_JSON-P_1.1.adoc]: This isn't totally correct, it does not work in most places as pointed out in https://github.com/wildfly/wildfly/blob/master/docs/src/main/asciidoc/Getting_Started_Guide.adoc#ee8-preview

Comment (rsvoboda) [reply ee/WFLY-6460_JSON-P_1.1.adoc]: FYI - https://github.com/wildfly/wildfly/pull/10886 to remove workaround from ee8-temp module

**Comment by jamezp** [comment]
Note added about the JMS client.

**Pull request merged**